gateway/src/apicast/policy/local_chain/local_chain.lua

```diff
@@ -45,6 +46,9 @@ local rewrite = _M.rewrite
 
 function _M:rewrite(context, ...)
   build_chain(context)
```

Comment:
Wouldn't it be nicer if `build_chain` would return the policy_chain object? Then we would not have to call `find_policy_chain` twice.

Reply:
You're right. It looks more readable. Changed in new commit.